java-logging-appinsights/src/main/resources/logback-includes.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does adding this here mean that other logback-includes won't be loaded?

Reply:
good question. unfortunately, it gets overwritten

Reply:
would it be worth adding another optional include to the base logback.xml for this?

Reply:
you're reading my mind 😃